### PR TITLE
First pass at working Performance helm definitions  

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

File: perf/create-secret-manager-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    vals:
+      - kubeSecretKey: datareposerviceaccount
+        path: secret/dsde/datarepo/perf/datarepo-sql-sa
+        encoding: base64
+        vaultKey: key
+  - secretName: database-pwd-perf
+    vals:
+      - kubeSecretKey: datarepopassword
+        path: secret/dsde/datarepo/perf/datarepo-sql-db
+        vaultKey: stairwaypassword
```
Comment (marikomedlock) [inline perf/create-secret-manager-secret.yaml:13]: datarepopassword?

Comment (marikomedlock) [inline perf/create-secret-manager-secret.yaml:13]: Should this be datarepopassword instead?

File: perf/datarepo-api.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+image:
+  tag: 68ec114-develop
+env:
+  GOOGLE_PROJECTID: broad-jade-perf
+  GOOGLE_SINGLEDATAPROJECTID: broad-jade-perf-data
+  DB_DATAREPO_USERNAME: drmanager
+  SPRING_PROFILES_ACTIVE: google,cloudsql,perf
```
Comment (marikomedlock) [inline perf/datarepo-api.yaml:8]: I think this should be "perftest" instead of "perf".

File: perf/create-secret-manager-secret.yaml (hunk 1 above)
Comment (fboulnois) [inline perf/create-secret-manager-secret.yaml:13]: I can't remember what we decided for the naming scheme.

Comment (smark88) [reply perf/create-secret-manager-secret.yaml:13]: so the key was the second one cause we only created one user, it's a slight bug

Comment (smark88) [reply perf/create-secret-manager-secret.yaml:13]: so the key was the second one cause we only created one user, it's a slight bug

**Comment by smark88** [comment]
merging for now can revisit later

**Pull request merged**